src/rp2_common/hardware_dma/include/hardware/dma.h

```diff
@@ -537,6 +537,38 @@ static inline void dma_set_irq1_channel_mask_enabled(uint32_t channel_mask, bool
     }
 }
 
+/*! \brief  Enable single DMA channel interrupt on either DMA_IRQ_0 or DMA_IRQ_1
+ *  \ingroup hardware_dma
+ *
+ * \param irq_index the IRQ index; either 0 or 1 for DMA_IRQ_0 or DMA_IRQ_1
+ * \param channel DMA channel
+ * \param enabled true to enable interrupt 0 on specified channel, false to disable.
```

Comment:
Should this be "enable interrupt irq_index on..." ?

Reply:
"to enable interrupt via irq_index for specified channel"